apps/web/middleware.ts

```diff
@@ -22,6 +22,11 @@ const safeGet = async <T = any>(key: string): Promise<T | undefined> => {
 const middleware = async (req: NextRequest): Promise<NextResponse<unknown>> => {
   const url = req.nextUrl;
   const requestHeaders = new Headers(req.headers);
+  const response = NextResponse.next();
+
+  if (response.headers.get("x-pages-router-error") === "true") {
+    return NextResponse.rewrite(new URL("/_not-found", req.url));
```

Comment:
I believe it is a page that doesn't exist actually. I tried to look it up in codebase.
Maybe we could make it more clear, like /_i_dont_exist

Reply:
<img width="742" alt="auth:login:123" src="https://github.com/user-attachments/assets/38bb28a7-b0ca-4e84-9091-fa1f778bb33d" />

`app/not-found.tsx

i think nextjs renders it implicitly in /_not-found